packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/simple_auth_configuration.only.bicep

```diff
@@ -0,0 +1,42 @@
+param simpleAuthWebAppName string
```

Comment:
This `simple_auth_configuration.only.bicep` file was removed from dev branch.

Reply:
Removed.